apache-pulsar/src/main/java/io/debezium/examples/apache/pulsar/config/PropertyLoader.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.examples.apache.pulsar.config;
+
+import org.apache.log4j.Logger;
+
+import java.util.Enumeration;
+import java.util.Properties;
+
+
+public class PropertyLoader {
+    private static Logger logger = Logger.getLogger(PropertyLoader.class);
+
+
+    public static void loadEnvironmentValues(Properties properties) {
+        Enumeration keys = properties.propertyNames();
+        while (keys.hasMoreElements()) {
+            String key = (String) keys.nextElement();
+            String env_safe_key = key.replace(".", "_");
```

Comment:
Please use Java naming confention

Reply:
env_safe_key can use concatenated calls of `replace` like `replace(".", "_").replace("-", "_"). Also char version of method might be better used then String arg version of `replace method

Reply:
Adjusted the name; I think the other stuff isn't too important for a demo.